.github/actions/monero-wallet-rpc/action.yml

```diff
@@ -0,0 +1,44 @@
+name: monero-wallet-rpc
+description: Spawns a Monero Wallet RPC Client
+
+inputs:
+  version:
+    description: "Version to download and run"
+    required: false
+    default: v0.18.1.2
+
+runs:
+  using: "composite"
+  steps:
+    - name: Monero Wallet RPC Cache
+      id: cache-monerowalletrpc
+      uses: actions/cache@v3
+      with:
+        path: monerowalletrpc
```

Comment:
This is the path of the file to cache. It should be monero-wallet-rpc accordingly.